tests/run/async_def.pyx

```diff
@@ -1,12 +1,13 @@
 # cython: language_level=3, binding=True
 # mode: run
-# tag: pep492, await, gh3337
+# tag: pep492, await, gh3337, gh2273
 
 """
 Cython specific tests in addition to "test_coroutines_pep492.pyx"
 (which is copied from CPython).
 """
 
+import asyncio
```

Comment:
There is a dedicated test file for the integration with `asyncio`, `py35_asyncio_async_def.srctree`. Please don't add the dependency here.